AdminServer/appscale/admin/service_manager.py

```diff
@@ -537,10 +570,12 @@ def _schedule_service(self, service_type, assignment_options):
       assignment_options: A dictionary specifying options
                           to use when starting servers.
     """
+    service = self.SERVICE_MAP[service_type]
     scheduled = [server for server in self.state
                  if server.type == service_type and
                  server.state in self.SCHEDULED_STATES]
-    to_start = assignment_options['count'] - len(scheduled)
+    default_count = service.default_count()
+    to_start = assignment_options.get('count', default_count) - len(scheduled)
```

Comment:
Is that for allowing manual configuration of count?

Reply:
Potentially manual (I did not test it), or a code change on the controller only.